### PR TITLE
More test coverage + Fix C_GetAttributeValue to set arg length when arg handle is NULL

### PR DESCRIPTION
**Pull request by dachalco**
Bump `iot_pcks11_mbedtls.c` to 97.3% line coverage.

`CKA_EC_PARAMS` attribute size can now be queried by passing `pValue = NULL`

### CODE REVIEW

File: source/portable/mbedtls/iot_pkcs11_mbedtls.c [hunk 1]
```diff
@@ -2849,10 +2849,11 @@ CK_DECLARE_FUNCTION( CK_RV, C_GetAttributeValue )( CK_SESSION_HANDLE hSession,
                     break;
 
                 case CKA_EC_PARAMS:
-
-                    pTemplate[ iAttrib ].ulValueLen = sizeof( ucP256Oid );
-
-                    if( pTemplate[ iAttrib ].pValue != NULL )
+                    if( pTemplate[ iAttrib ].pValue == NULL )
```
Comment (lundinc2) [inline source/portable/mbedtls/iot_pkcs11_mbedtls.c:2852]: Awesome catch !

**Review by lundinc2 — APPROVED** [review]

**Review by muneebahmed10 — APPROVED** [review]
